### ISSUE TITLE
portfolio: manage real variable as objective variable

### ISSUE THREAD

**Issue by pchtsp** [chocoteam/choco-solver#582]
## Expected behavior

Choco should be able to work with MPS files with multiple cores (`-p 3` option in choco-parsers).

## Actual behavior

The mps solver returns an error:

    c [-p, 3, /home/pchtsp/Documents/resources/choco-parsers/src/test/resources/mps/example1.mps]
    c 3 solvers in parallel
    c Unimodular: false
    c Unimodular: false
    c Unimodular: false
    Exception in thread "main" java.lang.UnsupportedOperationException: ParallelPortfolio cannot deal with real variable objective optimization problems
        at org.chocosolver.solver.ParallelPortfolio.check(ParallelPortfolio.java:412)
        at org.chocosolver.solver.ParallelPortfolio.prepare(ParallelPortfolio.java:254)
        at org.chocosolver.solver.ParallelPortfolio.solve(ParallelPortfolio.java:188)
        at org.chocosolver.parser.mps.MPS.manyThread(MPS.java:181)
        at org.chocosolver.parser.mps.MPS.solve(MPS.java:150)
        at org.chocosolver.parser.mps.ChocoMPS.main(ChocoMPS.java:27)
    s UNKNOWN

## Possible Solution

Check issue in choco-parsers: https://github.com/chocoteam/choco-parsers/issues/15

citing @cprudhom:

> That’s a limitation of the current portfolio, it doesn’t manage real variable as objective variable. That should be modified too. Can you create another issue on that point too (related to choco, not parser) ? Btw, thank you for the MWE.

## Steps to Reproduce (for bugs)

Using one of the mps examples. I even tried converting all the variables into integers in the mps and still got the error.

    java -cp .:choco-parsers-4.0.5-SNAPSHOT-with-dependencies.jar org.chocosolver.parser.mps.ChocoMPS -p 3 choco-parsers/src/test/resources/mps/example1.mps 

## Context

Trying to use choco with multiple processors to solve problems with the mps interface in choco-parsers.

## Environment

* Choco-solver version: `choco-solver-4.0.7-SNAPSHOT` + `choco-parsers-4.0.5-SNAPSHOT`
* Java version: `openjdk version "1.8.0_162"`
* PC details: I'm using Ubuntu 16.04 LTS 64 bits with an i7 processor and 16GB RAM.

**Comment by cprudhom** [comment]
Ibex is not threadsafe, so such a feature will only be available when no `RealConstraint` are declared (which may be the case for MPS files).

One things that needs to be done is to collect strategies over real variables to provide good variety and complementary of behaviors.
Any suggestions are welcomed.

**Comment by pchtsp** [comment]
What does it mean to not have RealConstraint?

As far as I tested the model used all integer variables and integer
coefficients and still got the error. Is there a way to make it work using
an MPS file that only implies editing the model?

F.

**Comment by cprudhom** [comment]
It means that no Ibex function is declared, only scalar constraints over real variables are used.
For choco-parser, when dealing with MPS files, the Ibex option has to be off (no declaration of `-ibex` in the CLI).

If there is a model with only integer (for variables and coefficients), then I would appreciate to have it to try to fix the issue.
